src/dom-utils/listScrollParents.js

```diff
@@ -12,8 +12,10 @@ export default function listScrollParents(
   const scrollParent = getScrollParent(element);
   const isBody = getNodeName(scrollParent) === 'body';
   const win = getWindow(scrollParent);
-  const target = isBody ? win : scrollParent;
-  const updatedList = list.concat(target, win.visualViewport || []);
+  const target = isBody
+    ? [].concat(win, win.visualViewport || [])
```

Comment:
```suggestion
    ? [win].concat(win.visualViewport || [])
```
this is a bit shorter